### ISSUE TITLE
Registry Edits and Outbound IP Connection

### ISSUE THREAD

**Issue by nexusits** [IndeedMiners/PolarisBiosEditor#7]
Writes the registry key hklm\software\microsoft\tracing\polarisbioseditor_rasapi32 with value: EnableFileTracing
Makes an Outbound connection to IP Address: 151.101.0.133

Labeled as Malicious by Bitdefender.

**Comment by IndeedMiners** [comment]
No idea what Bitdefender anti-virus is... but that IP is Github, just open the IP and you will see. It checks Github repo for new version of PBE. Please stop using these "anti-virus", use something like ESET.

https://github.com/IndeedMiners/PBE-Polaris-Bios-Editor-1.7.4/blob/master/PolarisBiosEditor.cs#L491

**Comment by nexusits** [comment]
BD GravityZone is an enterprise level cloud security platform. Similar to ESET's Endpoint Protection Advanced Cloud. I suspect with higher levels of Advanced Threat Detection(ATD), Managed Threat Response(MTR), and Endpoint Detection and Response(EDR) all products that implement those features would be picking these up.  

BDGZ only labeled as malicious due to the registry key edit that blocks process tracing. When I looked at the EDR logs, that's when I saw the IP connection. Figured I'd report the findings to ensure full transparency and help others understand whats going on should they see the same thing.
